model-optimizer/mo/moc_frontend/serialize.py

```diff
@@ -7,8 +7,8 @@
 from mo.back.ie_ir_ver_2.emitter import append_ir_info
 from mo.utils.cli_parser import get_meta_info, parse_transform
 
-from ngraph import Function         # pylint: disable=no-name-in-module,import-error
-from ngraph import function_to_cnn  # pylint: disable=no-name-in-module,import-error
+from openvino import Function         # pylint: disable=no-name-in-module,import-error
+from openvino import function_to_cnn  # pylint: disable=no-name-in-module,import-error
```

Comment:
I suppose `openvino` should not contain `CNNNetwork`

Reply:
removed